### PR TITLE
Replace RC-Progress Line with Material UI Progress Bar

### PR DESCRIPTION
**Pull request by Sonichigo**
## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by Sonichigo — COMMENTED** [review]
git commit -m 'signed-off-by : pathakanimesh12@gmail.com'

File: README.md [hunk 1]
```diff
@@ -60,7 +60,6 @@ Check out the <a href="https://github.com/litmuschaos/litmus/blob/master/ADOPTER
 Some of the considerations that need to be made with Litmus (as a chaos framework), are broadly listed here. Many of these are already being worked on
 as mentioned in the [ROADMAP](./ROADMAP.md). For details or limitations around specific experiments, refer to the respective [experiments docs](https://docs.litmuschaos.io/docs/pod-delete/).
```
Comment (S-ayanide) [inline README.md:62]: This point was removed from the README, which is not suggested

File: litmus-portal/cluster-agents/subscriber/pkg/gql/util.go [hunk 2]
```diff
@@ -24,7 +24,12 @@ func MarshalGQLData(gqlData interface{}) (string, error) {
 // generate gql mutation payload for workflow event
 func GenerateWorkflowPayload(cid, accessKey string, wfEvent types.WorkflowEvent) ([]byte, error) {
 	clusterID := `{cluster_id: \"` + cid + `\", access_key: \"` + accessKey + `\"}`
-	// process event data
+
+	for id, event := range wfEvent.Nodes {
+		event.Message = strings.Replace(event.Message, `"`, ``, -1)
+		wfEvent.Nodes[id] = event
+	}
+
```
Comment (S-ayanide) [inline litmus-portal/cluster-agents/subscriber/pkg/gql/util.go:32]: Can you please explain why this is necessary?

File: litmus-portal/frontend/src/components/ProgressBar/AnalyticsLinearProgressBar/index.tsx [hunk 3]
```diff
@@ -7,25 +7,30 @@ interface LinearProgressBarProps {
...
-      strokeColor={isInTable ? palette.primary.dark : palette.secondary.dark}
+      strokeColor={
+        isDefault
+          ? theme.palette.secondary.dark
+          : resultValue > 30 && resultValue <= 60
+          ? theme.palette.warning.main
+          : resultValue > 60
+          ? theme.palette.primary.dark
+          : theme.palette.error.dark
+      }
```
Comment (S-ayanide) [inline litmus-portal/frontend/src/components/ProgressBar/AnalyticsLinearProgressBar/index.tsx:33]: This change does not satisfy our requirement of changing `Line` to `ProgressBar`

File: litmus-portal/frontend/src/components/Targets/TargetCopy/index.tsx [hunk 4]
```diff
@@ -23,9 +23,13 @@ const TargetCopy: React.FC<InstallProps> = ({ yamlLink }) => {
       .writeText(text)
       .catch((err) => console.error('Async: Could not copy text: ', err));
 
-    setTimeout(() => setCopying(false), 3000);
+    setTimeout(() => setCopying(false), 6000);
```
Comment (S-ayanide) [inline litmus-portal/frontend/src/components/Targets/TargetCopy/index.tsx:26]: Is there any specific reason to change the interval of timeout?

File: translations/README-ja.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
+<img alt="LitmusChaos" src="https://landscape.cncf.io/logos/litmus.svg" width="200" align="left">
+
+# Litmus
+### クラウドネイティブ・カオスエンジニアリング
```
Comment (S-ayanide) [inline translations/README-ja.md:4]: We'd love to welcome this in our README, can you please send a separate PR only for this!

**Review by S-ayanide — CHANGES_REQUESTED** [review]

**Comment by S-ayanide** [comment]
Can you please take a look at the Frontend CircleCI build failure issues, mostly its about unused variables. Also could you please resolve the conflicts! Thanks

File: translations/README-ja.md (hunk 5 above)
Comment (Sonichigo) [reply translations/README-ja.md:4]: @S-ayanide Don't know how this happened but none of changes were made by me in my fork repo. I'll add this README in Japanese and German and send different PR.

**Comment by rajdas98** [comment]
Thanks, @Sonichigo for this PR. But this PR is pending for the last 6 months. Let us know if you want to work on this PR. Otherwise, we need to close this PR.
